src/transformers/models/auto/configuration_auto.py

```diff
@@ -35,6 +35,7 @@
         ("maskformer", "MaskFormerConfig"),
         ("decision_transformer", "DecisionTransformerConfig"),
         ("poolformer", "PoolFormerConfig"),
+        ("cvt", "CvtConfig"),
```

Comment:
Sorry to disappoint you, but the "v" comes after "o", so you can place "cvt" after "convnext" everywhere ;)

Reply:
Done